catkin_ws/src/simulation/worlds/quali.sdf

```diff
@@ -56,6 +56,64 @@
             </link>
         </model>
 
+          <model name="auv">
```

Comment:
this code is a duplicate of what is inside auv.sdf - include it similar to tethys/gate